src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -0,0 +1,1695 @@
+        {
+            metaData.bridgeOptions.isIMUEnabled = true;
+        }
+
+        if (ptr->getParameter("accelerometers_list", metaData.sensorsList.linearAccelerometersList, VectorResizeMode::Resizable))
+        {
+            metaData.bridgeOptions.isLinearAccelerometerEnabled = true;
+        }
+
+        if (ptr->getParameter("gyroscopes_list", metaData.sensorsList.gyroscopesList, VectorResizeMode::Resizable))
+        {
+            metaData.bridgeOptions.isGyroscopeEnabled = true;
+        }
+
+        if (ptr->getParameter("orientation_sensors_list", metaData.sensorsList.orientationSensorsList, VectorResizeMode::Resizable))
+        {
+            metaData.bridgeOptions.isOrientationSensorEnabled = true;
+        }
+
+        if (ptr->getParameter("magnetometers_list", metaData.sensorsList.magnetometersList, VectorResizeMode::Resizable))
+        {
+            metaData.bridgeOptions.isMagnetometerEnabled = true;
+        }
+
+        return true;
+    }
+
+    /**
+     * Configure six axis force torque sensors meta data
+     */
+    bool configureSixAxisForceTorqueSensors(std::weak_ptr<IParametersHandler> handler,
+                                            SensorBridgeMetaData& metaData)
+    {
+        constexpr std::string_view logPrefix = "[YarpSensorBridge::Impl::configureSixAxisForceTorqueSensors] ";
+        auto ptr = handler.lock();
+        if (ptr == nullptr) { return false; }
+
+        if (!ptr->getParameter("sixaxis_forcetorque_sensors_list", metaData.sensorsList.sixAxisForceTorqueSensorsList, VectorResizeMode::Resizable))
+        {
+            std::cerr << logPrefix << " Required parameter \"sixaxis_forcetorque_sensors_list\" not available in the configuration"
+                      << std::endl;
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Configure cartesian wrenches meta data
+     */
+    bool configureCartesianWrenches(std::weak_ptr<IParametersHandler> handler,
+                                    SensorBridgeMetaData& metaData)
+    {
+        constexpr std::string_view logPrefix = "[YarpSensorBridge::Impl::configureCartesianWrenchSensors] ";
+        auto ptr = handler.lock();
+        if (ptr == nullptr) { return false; }
+
+        if (!ptr->getParameter("cartesian_wrenches_list", metaData.sensorsList.cartesianWrenchesList, VectorResizeMode::Resizable))
+        {
+            std::cerr << logPrefix << " Required parameter \"cartesian_wrenches_list\" not available in the configuration"
+                      << std::endl;
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Configure cameras meta data
+     */
+    bool configureCameras(std::weak_ptr<IParametersHandler> handler,
+                          SensorBridgeMetaData& metaData)
+    {
+        constexpr std::string_view logPrefix = "[YarpSensorBridge::Impl::configureCameras] ";
+        auto ptr = handler.lock();
+        if (ptr == nullptr) { return false; }
```

Comment:
```suggestion
        if (ptr == nullptr) 
        {
             return false;
        }
```

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/106/commits/8ac1b32c085dd24fce06f38bb718677c1b4df29c